include/vctr/Containers/Vector.h

```diff
@@ -174,16 +174,30 @@ class Vector : public VctrBase<ElementType, std::vector<ElementType, Allocator<E
         return { Vctr::begin(), Vctr::end() };
     }
 
+    /** Standard interface function forwarded to std::vector. */
     constexpr void resize (size_t i) { Vctr::storage.resize (i); }
 
+    /** Standard interface function forwarded to std::vector. */
     constexpr void reserve (size_t i) { Vctr::storage.reserve (i); }
 
+    /** Standard interface function forwarded to std::vector. */
+    constexpr void shrink_to_fit() { Vctr::storage.shrink_to_fit(); }
+
+    /** Standard interface function forwarded to std::vector. */
     constexpr void push_back (ElementType&& newElement) { Vctr::storage.push_back (std::move (newElement)); }
 
+    /** Standard interface function forwarded to std::vector. */
     constexpr void push_back (const ElementType& newElement) { Vctr::storage.push_back (newElement); }
```

Comment:
The current `pago::Vector` implementation returns a reference to the last element instead of `void` which is quite handy in some situations. Could you add this feature here too?